src/main/java/jewellery/inventory/service/UserInOrganizationService.java

```diff
@@ -90,20 +97,7 @@ public void deleteUserInOrganization(UUID userId, UUID organizationId) {
         userId);
   }
 
-  private void validateCurrentUserPermission(
-      Organization organization, OrganizationPermission permission) {
-    User currentUser = userService.getUser(authService.getCurrentUser().getId());
-    if (!hasPermission(currentUser, organization, permission)) {
-      throw new MissingOrganizationPermissionException(
-          currentUser.getId(), organization.getId(), permission);
-    }
-    logger.debug(
-        "User permission validation successful. User ID: {}, Organization ID: {}",
-        currentUser.getId(),
-        organization.getId());
-  }
-
-  private void validateUserInOrganization(Organization organization) {
+  public void validateUserInOrganization(Organization organization) {
```

Comment:
Maybe we should move this to OrganizationService as well. Now that I think about it, it might make sense we keep both `validateUserInOrganization` and `validateCurrentUserPermission` to UserInOrganizationService. But regardless of where, i feel these should be kept in one place.

Reply:
Done!